api/types.go

```diff
@@ -6,22 +6,21 @@ import (
 	ma "github.com/multiformats/go-multiaddr"
 )
 
-type DealState int
+type DealState = uint64
```

Comment:
For enums it is better to use normal types instead of aliases.

Reply:
cbor-gen wasn't happy about that one for some reason